PLATFORM_SUPPORT.md

```diff
@@ -45,4 +45,4 @@ Other Tier 3 OS:
 
 * FreeBSD
 * VxWorks
-* QNX
\ No newline at end of file
+* QNX
```

Comment:
Since FastDDS is currently supporting Android as Tier3, should it be added as well?